src/vivarium_gates_nutrition_optimization/components/pregnancy.py

```diff
@@ -61,3 +141,29 @@ def Pregnancy():
         states=[not_pregnant, pregnant, postpartum],
         get_data_functions={"cause_specific_mortality_rate": lambda *_: 0.0},
     )
+
+
+def get_partial_term_probability_data(builder: Builder) -> pd.DataFrame:
+    asfr = builder.data.load(data_keys.PREGNANCY.ASFR).set_index(ARTIFACT_INDEX_COLUMNS)
+    sbr = (
+        builder.data.load(data_keys.PREGNANCY.SBR)
+        .set_index("year_start")
+        .drop(columns=["year_end"])
+        .reindex(asfr.index, level="year_start")
+    )
```

Comment:
What is the structure of SBR in the artifact that means you have to do this reshaping?

Reply:
SBR is not age specific, so it is essentially just a single value indexed by the year for a particular location. This reshaping is intended to extend it onto the relevant multiindex